CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
+## v2.3.2
+
+* MapboxCoreNavigation now requires [MapboxDirections v2.4._x_](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.4.0). ([#3846](https://github.com/mapbox/mapbox-navigation-ios/pull/3846))
```

Comment:
```suggestion

### Packaging

* MapboxCoreNavigation now requires [MapboxDirections v2.4._x_](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.4.0). ([#3846](https://github.com/mapbox/mapbox-navigation-ios/pull/3846))
```

Reply:
This wording nods to the fact that there was an incompatibility bug and we fixed it at the same time we increased the minimum requirement:

```suggestion
* Fixed MapboxCoreNavigation’s compatibility with [MapboxDirections v2.4._x_](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.4.0). ([#3846](https://github.com/mapbox/mapbox-navigation-ios/pull/3846))
```